### PR TITLE
GlotovES/hm3

### PR DESCRIPTION
**Pull request by GlotovES**

### CODE REVIEW

File: GlotovES/HM3/task_3.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+a = input("Enter a:= ")
```
Comment (kolyasalubov) [inline GlotovES/HM3/task_3.py:1]: good job

File: GlotovES/HM3/task_2.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+number = int(input("Input a four-digit natural number: "))
```
Comment (kolyasalubov) [inline GlotovES/HM3/task_2.py:1]: excellent work

File: GlotovES/HM3/task_1.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
+#####################################
+
+str_37 = 'If the implementation is easy to explain, it may be a good idea'
```
Comment (kolyasalubov) [inline GlotovES/HM3/task_1.py:3]: nice job, but you should have used the full python philosophy text

**Review by kolyasalubov — APPROVED** [review]

File: GlotovES/HM3/task_1.py (hunk 3 above)
Comment (GlotovES) [reply GlotovES/HM3/task_1.py:3]: Is it necessary to finish?

Comment (kolyasalubov) [reply GlotovES/HM3/task_1.py:3]: that would be a good finish this task
and also read carefully about the methods find() and count()

File: GlotovES/HM3/task_1.py [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+50.There should be one-- and preferably only one --obvious way to do it.
+51.Although that way may not be obvious at first unless you're Dutch.
+52.Now is better than never.
+53.Although never is often better than *right* now.
+54.If the implementation is hard to explain, it's a bad idea.
+55.If the implementation is easy to explain, it may be a good idea.
+56.Namespaces are one honking great idea -- let's do more of those!"""
+
+#####################################
+
```
Comment (kolyasalubov) [inline GlotovES/HM3/task_1.py:61]: good job

**Review by kolyasalubov — APPROVED** [review]

**Pull request merged**